python/activity_stream/ui/activity_stream_widget.py

```diff
@@ -1,44 +1,70 @@
 # -*- coding: utf-8 -*-
 
-# Form implementation generated from reading ui file 'activity_stream_widget.ui'
+# Form implementation ORIGINALLY generated from reading ui file 'activity_stream_widget.ui'
 #
 #      by: pyside-uic 0.2.13 running on PySide 1.1.1
 #
-# WARNING! All changes made in this file will be lost!
+# *******  NOT USING QT DESIGNER FOR THIS TEMPORARILY
```

Comment:
Not sure what the status of this is... just flagging it as part of code review to make sure there isn't some cleanup that should happen at this point...